scripts/fuse/elektra_fuse/elektra_fuse_interface.py

```diff
@@ -0,0 +1,218 @@
+#stdlib imports
+import errno, stat, time, re, psutil, logging, subprocess, os
+from pathlib import Path
+
+#fuse imports
+from fuse import FUSE, FuseOSError, Operations, LoggingMixIn
+
+#project imports
+from elektra_util import *
+import mock_context
+
+startup_time = time.time()
+
+#TODO: give appropriate error codes (instead of readonly fs) when non-authenticated user tries to do illegal things
+
+def getattr(path, fh=None):
+
```

Comment:
Can you write a (short) docu about what the functions are doing?

Reply:
Resolved in 530e3d46acc3efed9b01eaacfbb93ff1f43aa8a9